client/utils/index.js

```diff
@@ -247,3 +247,21 @@ export const objectRemoveEmptyProperties = ( obj ) => {
 		.filter( ( k ) => obj[ k ] !== null && obj[ k ] !== undefined )
 		.reduce( ( a, k ) => ( { ...a, [ k ]: obj[ k ] } ), {} );
 };
+
+/**
+ * Checks if the passed version is greater than base.
+ *
+ * @param {string} version Version that is compared.
+ * @param {string} base Version to compare with.
+ * @return {boolean} Whether version is greater than base.
+ */
+export const isVersionGreater = ( version, base ) => {
```

Comment:
A few suggestions for this function:

- I think it's more accurate to call it `isVersionGreaterOrEqual` - WDYT?
- Should we do some validation of the input strings? For example checking that they are in the format `x.y.z` where each number is between 1-9. Since it's an internal function I don't think we need to go too in detail with it, we could just update the docblock with examples of expected format and return false if it's invalid input.
- I wonder if we may see dev versions get passed in (like `9.9.0-beta`) - we could try and parse for that to make sure the function works in that case.

Reply:
Great suggestion! Fixed that and also added unit tests 👨‍🍳 https://github.com/Automattic/woocommerce-payments/pull/10706/commits/ea0b0bd4a5d4403715457c45c0334754dc8e339e

Reply:
Perfect! Great work 🧑‍🍳